package.json

```diff
@@ -80,6 +80,7 @@
     "typescript": "^4.9.5"
   },
   "jest": {
+    "maxWorkers": 1,
```

Comment:
This solves the issue with JEST not allow to debug issues with the test when they use assertions on native `BigInt` 

see https://stackoverflow.com/questions/75749980/typeerror-do-not-know-how-to-serialize-a-bigint

Reply:
I found a better solution! as this was a temporary fix, and didn't work in all cases!